src/bootstrap/flags.rs

```diff
@@ -18,7 +18,7 @@ use std::fs;
 use std::path::PathBuf;
 use std::process;
 
-use getopts::{Matches, Options};
+use getopts::{Options};
```

Comment:
Now that this line is just a single import, |use getopts::Options;| should also work.